src/ape/utils.py

```diff
@@ -386,14 +386,15 @@ class GithubClient:
     An HTTP client for the Github API.
     """
 
-    TOKEN_KEY = "GITHUB_ACCESS_TOKEN"
+    TOKEN_KEY_OPTIONS = ("GITHUB_ACCESS_TOKEN", "GH_ACCESS_TOKEN")
```

Comment:
had to do this because you cannot have env vars in github actions with the prefix `GITHUB_`